easybuild/easyblocks/s/superlu.py

```diff
@@ -105,16 +113,16 @@ def install_step(self):
         """
         super(EB_SuperLU, self).install_step()
 
-        self.libbits = 'lib'
-        if not os.path.exists(os.path.join(self.installdir, self.libbits)):
-            self.libbits = 'lib64'
+        libbits = 'lib'
```

Comment:
Doesn't this change behavior and potentially break the sanity check? Previously the sanity check did check lib64 (under the below condition) not only lib. So why change this?

Reply:
We have an automatic fallback mechanism in framework, whenever the sanity check can't find `lib/foo`, it will automatically also check for `lib64/foo`, and pass the check if the latter is found, see https://github.com/easybuilders/easybuild-framework/pull/2477